app/views/individual/matching/NotEligibleForIncomeTax.scala.html

```diff
@@ -34,10 +36,11 @@
 
     <h1 class="govuk-heading-l">@messages("cannot-sign-up-yet.heading")</h1>
 
-   <p class="govuk-body">
-       <a class="govuk-link" href="https://www.gov.uk/guidance/check-if-youre-eligible-for-making-tax-digital-for-income-tax">
-           @hmrcNewTabLinkHelper(NewTabLinkHelper(messages("cannot-sign-up-yet.para1")))
-       </a>
+   <p class="govuk-body" id="cannot-sign-up-yet-link">
+       @Html(messages(
+       key = "cannot-sign-up-yet.para1",
+       args = hmrcNewTabLinkHelper(NewTabLinkHelper(messages("cannot-sign-up-yet.link"), Some(appConfig.govukGuidanceITSAQualifyingIncomeLink)))
+       ))
```

Comment:
```suggestion
       @hmrcNewTabLinkHelper(NewTabLinkHelper(messages("cannot-sign-up-yet.link"), Some(appConfig.govukGuidanceITSAQualifyingIncomeLink)))
```
Should work correctly, no need for the message with only a parameter